charts/kusion/README.md

```diff
@@ -1,6 +1,6 @@
 # Kusion Chart
```

Comment:
The AppVersion badge still shows 0.14.0 while the Version badge was updated to 0.14.2. These versions should be consistent. Please update the AppVersion badge to match the new version.